WISE-Community/WISE
# Implemented timeline view for students

Implemented timeline view for student runs on the student homepage. Also updated styles so that student run list items and teacher run list items match. Improved the mobile layout for run list items.

Closes issue #1437.

To test:
- Log in as a student and make sure the student run list now uses the timeline view.
- Check layout on different size screens from mobile to desktop.
- Make sure search works correctly (and that searching by run owner/teacher also works.
- Log in as a teacher and check Class Schedule layout on different screen sizes just to make sure everything looks okay.
